src/XMakeBuildEngine/UnitTests/Evaluation/Expander_Tests.cs

```diff
@@ -3280,7 +3280,10 @@ public void Medley()
                 "$(e.Substring($([System.IO.Path]::Combine(`a`, `b`))))",
                 "$([]::())",
                 "$((((",
+#if FEATURE_WIN32_REGISTRY
+                // If no registry, this gets expanded to the empty string
                 "$(Registry:X)",
+#endif
```

Comment:
Doesn't work well here because it's an array (so fixed length)--what should be put in its place? I thought about adding logic in the loop that consumes the list but that got complicated.

A better solution here would be to convert this to an xunit `Theory` data-driven test and avoid that entry via an IsWindows check. But I'm trying to _reduce_ merge conflicts here :)